package.json

```diff
@@ -47,6 +47,7 @@
 	},
 	"xo": {
 		"rules": {
+			"@typescript-eslint/no-extraneous-class": "off",
```

Comment:
This change is not related to this PR, but some dependencies of xo updated automatically caused existing code checks to fail